src/components/ComponentExample.tsx

```diff
@@ -14,7 +14,8 @@ const ComponentExample = ({ style, children }: Props): JSX.Element => (
       background:
         style && style.background
           ? style.background
-          : suomifiTheme.colors.depthLight30,
+          : suomifiTheme.colors.whiteBase,
+      border: style.border || `1px solid ${suomifiTheme.colors.depthBase}`,
```

Comment:
When tried to run this branch I got the following error:
```
TypeError: Cannot read property 'border' of undefined
>  border: style.border || `1px solid ${suomifiTheme.colors.depthBase}`,
```

You might want to check if the `style` and `border` exists, like with the `background`

Reply:
Ah, the check itself produces the right result, but before resolving that style.border is undefined it produces that error as well. I'll change it.

Reply:
Fixed.